### PR TITLE
Test pandas 0.20.1

### PR DESCRIPTION
**Pull request by jcrist**
Add new pandas release to travis and appveyor.

### CODE REVIEW

**Comment by jcrist** [comment]
I reverted the change to test 0.20.0 on appveyor. The recent pandas release isn't in defaults yet, and switching to conda-forge revealed a bad windows build for qt. Easier just to wait until those issues are resolved.

**Comment by jcrist** [comment]
Waiting on the 0.20.1 build to finish (and update to that) before merging: https://github.com/conda-forge/pandas-feedstock/pull/24.

File: .travis.yml [hunk 1]
```diff
@@ -5,10 +5,10 @@ os: linux
 env:
   matrix:
     - PYTHON=2.7 NUMPY=1.11.3 PANDAS=0.19.0 COVERAGE='true' PARALLEL='false' TEST_IMPORTS='false' XTRATESTARGS=
-    - PYTHON=2.7 NUMPY=1.12.1 PANDAS=0.19.2 COVERAGE='false' PARALLEL='true' TEST_IMPORTS='false' PYTHONOPTIMIZE=2 XTRATESTARGS=--ignore=dask/diagnostics
+    - PYTHON=2.7 NUMPY=1.12.1 PANDAS=0.20.0 COVERAGE='false' PARALLEL='true' TEST_IMPORTS='false' PYTHONOPTIMIZE=2 XTRATESTARGS=--ignore=dask/diagnostics
```
Comment (TomAugspurger) [inline .travis.yml]: Bump these to 0.20.1 I think.

Comment (jcrist) [inline .travis.yml]: Waiting on the build to finish, 0.20.1 isn't quite available yet :)

**Pull request merged**